src/pages/task/actionButtons/previousCommits/PreviousCommits.tsx

```diff
@@ -152,6 +152,8 @@ export const PreviousCommits: React.VFC<PreviousCommitsProps> = ({
     }
   }, [shouldFetchLastExecuted]); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const linkComp = useLGButtonRouterLink(link);
+
   return (
     <PreviousCommitsWrapper>
```

Comment:
I think we need to tweak the alignment for this component!

<img width="360" alt="Screen Shot 2022-11-22 at 4 52 48 PM" src="https://user-images.githubusercontent.com/47064971/203429887-07d139ad-de60-46e7-b536-b8d14d6d6fa3.png">